src/FlashMessages.php

```diff
@@ -161,11 +165,17 @@ public function prepareMessages(SessionInterface $session, string $sessionKey):
             return;
         }
 
-        $sessionMessages = $session->get($sessionKey);
-        $sessionMessages = ! is_array($sessionMessages) ? [] : $sessionMessages;
+        $sessionMessages = $this->getStoredMessages($sessionKey);
 
+        /** @var array<string,mixed> $currentMessages */
```

Comment:
Are messages generally `mixed`?

Reply:
While I just store strings there like a normal person, mezzio-flash doesn't impose any limitations. It's possible some madman is out there abusing that fact mightily :-)